data/gradleFiles/dependenciesData.gradle

```diff
@@ -0,0 +1,7 @@
+ext {
+    kotlin_version = '1.3.31'
```

Comment:
@OscarGonzalezLesaga I think that we can remove this kotlin_version line, because it is already defined in the project build.gradle. What do you think?

Reply:
For me is better to set it in this file too to maintain the dependencies in all modules